app/jobs/notify_web_hook_job.rb

```diff
@@ -0,0 +1,50 @@
+class NotifyWebHookJob < ApplicationJob
+  extend StatsD::Instrument
+  include TraceTagger
+
+  queue_as :default
+  self.queue_adapter = :good_job
+
+  before_perform { @kwargs = arguments.last.then { _1 if Hash.ruby2_keywords_hash?(_1) } }
+  before_perform { @webhook = @kwargs.fetch(:webhook) }
+  before_perform { @protocol = @kwargs.fetch(:protocol) }
+  before_perform { @host_with_port = @kwargs.fetch(:host_with_port) }
+  before_perform { @version = @kwargs.fetch(:version) }
+  before_perform { @rubygem = @version.rubygem }
+
+  attr_reader :webhook, :protocol, :host_with_port, :version, :rubygem
+
+  def perform(*)
+    url = webhook.url
+    set_tag "gemcutter.notifier.url", url
+    set_tag "gemcutter.notifier.webhook_id", webhook.id
+
+    timeout(5) do
```

Comment:
Super minor idea: What about to wrap timeout into constant since it is used 3 time?

Reply:
will probably be replacing this code soon anyways

Reply:
OK, let's keep it as is for now.